lib/metadata/src/metadata.cpp

```diff
@@ -0,0 +1,231 @@
+#include "irods/metadata.hpp"
+#include "irods/filesystem.hpp"
```

Comment:
Add an empty line before this.

This lets the reader know which header represents the primary header.

Reply:
I've never thought of this as a best practice but I will now follow it religiously.

Reply:
I don't know if it is a best practice. To me, anything that helps the reader not feel overwhelmed is good.